packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -650,12 +644,7 @@ export function TokenSearch({
         <div className="flex justify-end pt-6">
           {isFetchingTokenLists ? (
             <span className="flex flex-row items-center gap-2 text-sm font-normal text-gray-9">
-              <Loader
-                type="Oval"
-                color="rgb(40, 160, 240)"
-                height={16}
-                width={16}
-              />
+              <Loader color="#28A0F0" size="small" />
```

Comment:
when do we decide to use `size="small"` or leave it empty?
I see that the default is small

Reply:
IMHO it's always good to explicitly mention the `size` - that's why I haven't left it empty anywhere - and it should be the standard.
Hence removing the default condition now. It will throw an error if the size isn't stated.